Makefile

```diff
@@ -45,16 +48,70 @@ server-shell: compose-build
 	@docker-compose run $(IMAGE_NAME) /bin/bash
 
 integration-shell: compose-build
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml run $(IMAGE_NAME) /bin/bash
+ifeq (,$(env))
+	@echo "no environment specified"
+	@path=""; \
+	for word in $(DOCKERFILE_ENVIRONMENTS); do \
+		path="$$path -f docker-compose.integration-$$word.yml"; \
+	done; \
+	docker-compose -f docker-compose.yml $$path build; \
+	docker-compose -f docker-compose.yml $$path up -d; \
+	sleep 5; \
+	setup_path=""; \
+	for word in $(DOCKERFILE_ENVIRONMENTS); do \
+		setup_path="fidesops python tests/integration_tests/$$word-setup.py"; \
```

Comment:
These setup files need to be hyphen delineated because bash doesn't treat the underscore delineated equivalent as a variable substitution.

e.g. `setup_path="fidesops python tests/integration_tests/$$word_setup.py"; \` translates to bath looking for the file `tests/integration_tests/.py`, whereas `setup_path="fidesops python tests/integration_tests/$$word-setup.py"` substitutes `$$word` correctly into `tests/integration_tests/env-setup.py`

Reply:
I know this is a bit late to the game, but have you considered just forklifting this logic to a python script instead that exec's the ultimate `docker` command?

The translation would be pretty straightforward and you could use Python instead of bash, which is:
1. easier to read
2. supported on windows

...?

Reply:
See `make docs-build` where I already broke the Makefile boundary and added a `generate_openapi.py` script for a little task like this...

Reply:
Will take a look, thanks!